runtime/lua/vim/lsp/diagnostic.lua

```diff
@@ -692,7 +730,9 @@ end
 -- send diagnostic information and the client will still process it. The
 -- diagnostics are simply not displayed to the user.
 function M.disable(bufnr, client_id)
-  vim.notify_once('vim.lsp.diagnostic.disable is deprecated. See :h deprecated', vim.log.levels.WARN)
+  local message = 'vim.lsp.diagnostic.disable is deprecated. See :h deprecated'
+  message = message .. '\nThis function will be removed in version 0.8.'
```

Comment:
We probably want a `vim.deprecate()` function that standardizes this pattern.

```
@param name of the deprecated thing
@param alternative thing to use instead of `name`
@param version Nvim that will remove `name`
function deprecate(name, alternative, version)
```

Reply:
Accidentally squashed.

Anyway, it's done. Please take a look.